rita/src/exit.rs

```diff
@@ -1,3 +1,14 @@
+//! This is the main source file for the Rita exit binary, by 'exit' we mean 'a vpn server, not a
+//! mesh router out in the field'.
+//!
+//! All meshing and billing functionaltiy is contained in rita_common and is common to both rita and
+//! rita_exit. The major difference is billing and connection code for the 'exit', the mandatory
```

Comment:
should be traffic counting that is different, to distinguish from "billing functionality" which you say is common